test/e2e/apimachinery/resource_quota.go

```diff
@@ -335,6 +335,7 @@ var _ = SIGDescribe("ResourceQuota", func() {
 		ginkgo.By("Creating a ResourceQuota")
 		quotaName := "test-quota"
 		resourceQuota := newTestResourceQuota(quotaName)
+		resourceQuota.Spec.Hard[v1.ResourceConfigMaps] = resource.MustParse(hardConfigMaps)
```

Comment:
I checked that only this test case creates configmap.

As you mentioned, it may be related to the CA configmap created by default in 1.20. Changing it to 10 configmaps is acceptable.

I wondered that if we poll inside that method, it will have some side effects like performance issues or flakes? (Let me refactor it if this is considered to be a better solution.)

So I prefer to update it to 10.

Reply:
loosening to 10 makes sense to me

Reply:
Updated. Thanks.